libs/angular-components/community/components/overlay/modal/footer/modal-footer.component.html

```diff
@@ -0,0 +1,32 @@
+<div
+  *ngIf="buttons() && buttons().length > 0"
```

Comment:
this check shouldn't be necessary. the way i see it, most of use cases will use ng-content instead of the buttons input